confidential-data-hub/docs/use-cases/secure-mount-with-block-device.md

```diff
@@ -75,3 +79,116 @@ $ lsblk |grep "encrypted_disk"
 └─encrypted_disk_OEyEj_dif 253:1    0 968.6M  0 crypt
   └─encrypted_disk_OEyEj   253:2    0 968.6M  0 crypt /mnt/test-path
 ```
+
+### Best Practices
+
+#### Ephemeral Storage
+
+Ephemeral storage is a storage that is not persistent.
+
+1. Mount a block device file, encrypt it using `luks2` with an ephemeral key, and mount the plaintext device to `/mnt/dev-path` as a file.
```

Comment:
You might want to make it clear that these are two different options not two steps.

Reply:
Fixed the document due to new API design. ptal